components/layout_2020/Cargo.toml

```diff
@@ -49,6 +49,8 @@ servo_geometry = { path = "../geometry" }
 servo_url = { path = "../url" }
 style = { workspace = true }
 style_traits = { workspace = true }
+taffy = { workspace = true }
+stylo_taffy = { workspace = true }
```

Comment:
Please integrate these conversions directly into the `layout_2020` crate instead making them a separate crate.